catalog/core/catalog-core-solr/src/test/java/ddf/catalog/source/solr/RemoteSolrCatalogProviderTest.java

```diff
@@ -80,12 +74,16 @@ public void testNullFilterAdapator() {
         null,
         mock(SolrClient.class),
         mock(SolrFilterDelegateFactory.class),
-        mock(DynamicSchemaResolver.class)) {
-      @Override
-      protected Future<SolrClient> createClient() {
-        return null;
-      }
-    };
+        mock(DynamicSchemaResolver.class)) {};
+  }
+
+  @Test(expected = IllegalArgumentException.class)
+  public void testNullCl() {
```

Comment:
✏️ `testNullSolrClient()`?

Reply:
ouch! I know I am cryptic but that is way more than what I usually do 😉